ABCDEnvironment/ABCDEnvironment.py

```diff
@@ -0,0 +1,72 @@
+import importlib.metadata
+import importlib.resources
+import importlib.util
+
+import slicer
+from slicer.ScriptedLoadableModule import (
+    ScriptedLoadableModuleWidget,
+    ScriptedLoadableModule,
+)
+from slicer.i18n import tr as _
+from slicer.i18n import translate
+from slicer.util import VTKObservationMixin
+
+
+def resolve_dependencies():
+    requirements = importlib.resources.files("LibABCDEnvironment") / "requirements.txt"
+    slicer.util.pip_install(["-r", str(requirements)])
```

Comment:
I like this approach with `importlib.resources`! Never seen this before. I like the way you put a requirements.txt as a resource under `LibABCDEnvironment` :+1: :+1: :+1:

Reply:
I make it a habit because it works nicely with `pip` installations and zip imports. Although neither of those apply to Slicer (currently) I think it's a good habit anyway.